instrumentation/opentelemetry-instrumentation-django/tests/test_middleware_asgi.py

```diff
@@ -59,7 +59,7 @@
 if DJANGO_2_0:
     from django.urls import re_path
 else:
-    from django.conf.urls import url as re_path
+    from django.conf.urls import url as re_path  # pylint: disable=E0611
```

Comment:
Added this as django.conf.urls.url() is removed in django 4.0. leading to lint check failure.